Dockerfile

```diff
@@ -493,6 +499,9 @@ ENV GEM_HOME=/var/tmp/gem
 ENV PATH=$PATH:$RBENV_ROOT/bin:$RBENV_ROOT/shims:$RBENV_ROOT/plugins/ruby-install/bin
 COPY --from=ruby --chown=$USER:$USER $RBENV_ROOT $RBENV_ROOT
 
+# ScanCode license texts directory
+COPY --from=scancode-license-data --chown=$USER:$USER /opt/scancode-license-data /opt/scancode-license-data
```

Comment:
Why did you remove the previous removal of unneeded files? Can we maybe elegantly replace that with only copying `/opt/scancode-license-data/*.LICENSE` here?

Reply:
Ah, I missed [your comment](https://github.com/oss-review-toolkit/ort/pull/9622#discussion_r1891394271) as you already resolved / collapsed the conversation. So, still, what do you think about just copying needed files here?

Reply:
Copying the needed stuff is more complicated than expected, so I went back deleting the files we don't need.

Reply:
> Copying the needed stuff is more complicated than expected

Oh, interesting, mind sharing the details? Why does

    COPY --from=scancode-license-data --chown=$USER:$USER /opt/scancode-license-data/*.LICENSE /opt/scancode-license-data

not work?

> I went back deleting the files we don't need.

Unfortunately, that now seems to miss deleting the `static` directory, or?

Reply:
Ping @wkl3nk just in case you overlooked this comment.